README.md

```diff
@@ -302,13 +302,15 @@ Hurray! You successfully made a contribution!
   - [Jayvir Rathi](https://github.com/Jayvirrathi)
   - [Jessica Salbert](https://github.com/jessicasalbert)
   - [Jivthesh](https://github.com/jivthesh)
+  - [John Daniel](https://github.com/johndanny98)
```

Comment:
Please add your name in line 306.

Reply:
@johndanny98 sorry, didn't realise that it is meant to be on line 306!

Reply:
I had added my name to line 306 and removed the extra space on the bottom and commited changes to this branch again